### ISSUE TITLE
Rüssi Schmied Quest "Habt ihr das verloren" aktiviert sich nicht

### ISSUE THREAD

**Issue by xTyraelx** [ClassicWoW/Nefarian_1.12.1_Bugtracker#2461]
Welches Verhalten wird beobachtet?

http://wowdata.buffed.de/quest/Habt-Ihr-das-verloren-3321

Quest steht nichz zur Auswahl

Wie sollte es sich verhalten?

Quest sollte zur Auswahl stehen. NPC sollte dabei ein Trinket herstellen.

Schritte zur Reproduzierung

Habe alle Rüssi Schmieds Quest erledigt

Zusätzliche Informationen (Screenshots, Videos, Klasse, Rasse, Level, etc.)

Level 47 derzeit, Untoter Krieger..Quest steht nicht zur AUswahl

Alle Kreaturen, Items, Objekte, Quests, Zauber etc. müssen aus unserer Datenbank verlinkt sein.

http://wowdata.buffed.de/quest/Habt-Ihr-das-verloren-3321

**Comment by hraesvelgr** [comment]
https://github.com/ClassicWoW/Nefarian_1.12.1_Bugtracker/issues/633 

bist du dir sicher, dass du alle prequests abgeschlossen hast?

**Comment by xTyraelx** [comment]
Ja, ich kann alles herstellen. Alle verschnörkelten Sachen. Sonst hätte ich nie Rüssi Schmied werden können.
